Library/Homebrew/requirements.rb

```diff
@@ -114,10 +114,15 @@ def message
   end
 end
 
+class CVSRequirement < Requirement
+  fatal true
+  default_formula "cvs"
```

Comment:
This isn't going to work as it's going to mean that bottled formulae always use the CVS formula rather than the system version.